packages/main/src/RPA/Crypto.py

```diff
@@ -104,9 +119,19 @@ def generate_key(self) -> str:
         If the key is lost, the encrypted data can not be recovered.
         If anyone else gains access to it, they can decrypt your data.
         """
-        return Fernet.generate_key().decode("utf-8")
+        if encryption_type == EncryptionType.FERNET:
+            return Fernet.generate_key().decode("utf-8")
+        elif encryption_type == EncryptionType.AES256:
+            key = os.urandom(32)
```

Comment:
Although `os.urandom` result "should" be good enough for cryptographic operations, i'd be best to use a dedicated cryptographically strong random source.

For example secrets library added in Python 3.6  https://docs.python.org/3/library/secrets.html seems like a good choice for this purpose?